autogpt_platform/backend/backend/copilot/model_router.py

```
@@ -291,3 +324,114 @@ async def resolve_model_route(
         return ResolvedModel(cell_slug, "catalog")
 
     return await _env_floor(config, mode, tier)
+
+
+async def resolve_codex_model_route(
+    mode: ModelMode,
+    tier: ModelTier,
+    credential_lease: "CredentialLease | PooledCodexRuntimeLease",
```

Ruff (0.16.1)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)